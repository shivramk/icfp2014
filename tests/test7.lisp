(define (main) (+ 4 (/ 15 3)))
