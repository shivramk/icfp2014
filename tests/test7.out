LDC 4
LDC 15
LDC 3
DIV
ADD
